StoreTestWPF.ViewModel/ViewModels/ModifyCakeViewModel.cs

```diff
@@ -0,0 +1,70 @@
+﻿using StoreTestWPF.DAL.Models;
+using StoreTestWPF.ViewModel.Interfaces;
+using StoreTestWPF.ViewModel.Utils;
+using System.Windows.Input;
+
+namespace StoreTestWPF.ViewModel.ViewModels
+{
+    public sealed class ModifyCakeViewModel : ViewModelBase
+    {
+        private CakeViewModel modifiedCake;
+        private ICommand acceptCommand;
+        private ICommand openFileCommand;
+        private IViewService viewService;
+
+        public string WindowName { get; set; }
+
+        public ModifyCakeViewModel(IViewService viewService)
+        {
+            this.viewService = viewService;
+        }
+
+        public CakeViewModel ModifiedCake
+        {
+            get { return this.modifiedCake; }
+            set
+            {
+                if (this.modifiedCake != value)
+                {
+                    this.modifiedCake = value;
+                    OnPropertyChanged(nameof(this.ModifiedCake));
+                }
+            }
+        }
+
+        public ICommand OpenFileCommand
```

Comment:
Lets use:
public ICommand OpenFileCommand => this.openFileCommand ?? new RelayCommand(this.OpenFileExecuted);

Reply:
Please change in all places.

Reply:
Fixed